tests/openthread_ncp/main.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (C) 2017 Baptiste CLENET
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       OpenThread test application
+ *
+ * @author      Baptiste Clenet <bapclenet@gmail.com>
+ */
+
+#include <stdio.h>
+
+#include "ot.h"
+
+int main(void)
+{
+    /* Run wpantund to interact with NCP */
```

Comment:
openthread_ncp aims to be connected to wpantund so the welcome message appears when you run it and wpantund is connected to by UART, No need to write something in RIOT main